era/base.js

```diff
@@ -107,16 +160,32 @@ export const threatFunctions = {
     extraCoeff = 1
   ) {
     // extraCoeff is only used for tooltip text
-    let a = fight.eventToUnit(ev, "source");
-    let b = fight.eventToUnit(ev, "target");
-    if (!a || !b) return;
-    let coeff = (useThreatCoeffs ? a.threatCoeff(ev.ability) : 1) * extraCoeff;
-    b.addThreat(a.key, amount, ev.timestamp, ev.ability.name, coeff);
+    let source = fight.eventToUnit(ev, "source");
+    let target = fight.eventToUnit(ev, "target");
+    if (!source || !target) return;
+    let coeff = applyThreatCoefficient(
+      useThreatCoeffs ? source.threatCoeff(ev.ability) : BASE_COEFFICIENT,
+      extraCoeff,
+      ev.ability.name
+    );
+    target.addThreat(source.key, amount, ev.timestamp, ev.ability.name, coeff);
   },
+
+  /**
+   * @param {import("./threat/wcl.js").WCLEvent} ev
+   * @param {import("./threat/fight.js").UnitSpecifier} unit
+   * @param {Fight} fight
+   * @param {number} amount
+   * @param {boolean} [useThreatCoeffs]
+   */
   unitThreatenEnemiesSplit(ev, unit, fight, amount, useThreatCoeffs = true) {
     let u = fight.eventToUnit(ev, unit);
     if (!u) return;
-    let coeff = useThreatCoeffs ? u.threatCoeff(ev.ability) : 1;
+    let coeff = applyThreatCoefficient(
+      useThreatCoeffs ? u.threatCoeff(ev.ability) : BASE_COEFFICIENT,
```

Comment:
Same here, for split coeff, we just add a note so it's clear where it came from.